pkg/plugins/policies/meshtrace/api/v1alpha1/meshtrace.go

```diff
@@ -34,12 +34,20 @@ type Conf struct {
 	Tags *[]Tag `json:"tags,omitempty"`
 }
 
-// Only one of zipkin or datadog can be used.
+// Only one of zipkin, datadog or opentelemetry can be used.
 type Backend struct {
 	// Zipkin backend configuration.
 	Zipkin *ZipkinBackend `json:"zipkin,omitempty"`
 	// Datadog backend configuration.
 	Datadog *DatadogBackend `json:"datadog,omitempty"`
+	// OpenTelemetry backend configuration.
+	OpenTelemetry *OpenTelemetryBackend `json:"opentelemetry,omitempty"`
+}
+
+// OpenTelemetry tracing backend configuration.
+type OpenTelemetryBackend struct {
+	// Address of OpenTelemetry collector e.g. otelcol:4317.
+	Endpoint string `json:"url"`
```

Comment:
Ups, good catch. Y, but it would recommend going with `endpoint` in both cases. 
Compared to the zipkin url it has no scheme and path, an example value would be `dest:1234`.

Reply:
Ok 👍!